src/components/molecules/CoreTable/CoreTable.tsx

```diff
@@ -0,0 +1,262 @@
+                'border-t border-[#f0f0f0]',
+                index % 2 === 0 ? 'bg-[#FAFAFA]' : 'bg-[#FFFFFF]'
+            )}
+        >
+            <td className='min-w-36 p-4 '>
+                <div className='flex items-center gap-3 laptop:justify-center'>
+                    <Tooltip
+                        iconElement={
+                            <span className='inline-block w-[100px] font-numerical'>
+                                {AddressUtils.format(trove.ownerAddress, 6)}
+                            </span>
+                        }
+                        placement='top'
+                    >
+                        {trove.ownerAddress}
+                    </Tooltip>
+
+                    <button onClick={() => setCopied(trove.ownerAddress)}>
+                        {copied === trove.ownerAddress ? (
+                            <CheckIcon className='h-4 w-4 text-success-700' />
+                        ) : (
+                            <Copy className='h-4 w-4 text-gray-400' />
+                        )}
+                    </button>
+                </div>
+            </td>
+            <td className='min-w-30 p-4 text-center'>
+                {trove.collateral.prettify()}
+            </td>
+            <td className='min-w-30 p-4 text-center'>
+                {trove.debt.prettify()}
+            </td>
+            <td className='min-w-30 p-4 text-center'>
+                <CollateralRatioBadge ratio={calculateCollateralRatio(trove)} />
+            </td>
+            <td className='min-w-30 p-4 text-center'>
+                {trove.debtInFront?.prettify()}
+            </td>
+            <td className='min-w-[180px] p-2'>
+                <div className='relative h-10 w-full'>
+                    <Transaction
+                        id={txId}
+                        send={async overrides => {
+                            setCurrentTxId(txId);
+                            try {
+                                return await sfStablecoin.send.liquidate(
+                                    trove.ownerAddress,
+                                    overrides
+                                );
+                            } finally {
+                                setCurrentTxId(null);
+                            }
+                        }}
+                    >
+                        <button
+                            className='h-10 w-full truncate rounded-xl border border-[#E3E3E3] bg-[#FAFAFA] px-3 py-1.5 text-sm font-medium text-[#565656] hover:bg-[#F0F0F0] hover:text-[#333333] disabled:cursor-not-allowed disabled:border-[#E3E3E3] disabled:bg-[#F7F7F7] disabled:text-[#B0B0B0]'
+                            disabled={
+                                !isConnected ||
+                                (currentTxId !== null && currentTxId !== txId)
+                            }
+                        >
+                            Liquidate Trove
+                        </button>
```

Comment:
The Button component should be used here for consistency. If the style is different with the current Button component, it's the design mistake.